hbase_storage/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java

```diff
@@ -54,9 +56,25 @@ public HPartitionCreator(HBaseTableInfoFactory tableInfoFactory,Connection conne
 
     @Override
     public PartitionCreator withName(String name){
+        return withName(name, Conglomerate.Priority.NORMAL);
+    }
+
+    public static int GetHBasePriority(Conglomerate.Priority priority)
+    {
+        switch(priority){
+            case NORMAL:    return HConstants.NORMAL_QOS;
+            case HIGH:      return HConstants.ADMIN_QOS;
+            case VERY_HIGH: return HConstants.HIGH_QOS;
```

Comment:
Nitpick: I would not even include VERY_HIGH since it maps to something that may be dangerous to use. (HIGH_QOS being even greater than SYSTEMTABLE_QOS that is used for HBASE system tables)

Reply:
it's not higher, it's equal. Still, i can remove.
`public static final int SYSTEMTABLE_QOS = HIGH_QOS;`